tests/test_format.py

```diff
@@ -225,26 +225,27 @@ def test_format_comparison_bars_to_js_exceptions(self):
             self.comparison_coords_headers, self.comparison_coords_data, 11)
 
     def test_format_emperor_html_footer_string(self):
+        self.maxDiff=5000
         """Test correct formatting of the footer string"""
         # footer for a jackknifed pcoa plot without biplots
         out_string = format_emperor_html_footer_string(False, True)
-        self.assertEqual(out_string, EXPECTED_FOOTER_A)
+        self.assertItemsEqual(out_string.split('/'), EXPECTED_FOOTER_A.split('/'))
```

Comment:
I would prefer '\n' then. Also, since assertItemsEqual doesn't care
about the order of the elements, we would need to still have the
self.assertEqual check. The first would help developers identify the
lines that need to be updated and the second verifies there's a
consistently and correctly structured HTML string.

On (Mar-04-15|15:45), mortonjt wrote:
> >          """Test correct formatting of the footer string"""
> >          # footer for a jackknifed pcoa plot without biplots
> >          out_string = format_emperor_html_footer_string(False, True)
> > -        self.assertEqual(out_string, EXPECTED_FOOTER_A)
> > +        self.assertItemsEqual(out_string.split('/'), EXPECTED_FOOTER_A.split('/'))
> 
> Just a random common letter ('/' occurs before every major html header).  I think '\n' would work just as well.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/352/files#r25826838